src/middleware/web-worker/index.ts

```diff
@@ -32,7 +38,7 @@ export function createWebWorkerHandler<T extends Options<ClientType>>(
     );
     return octokitResponse
       ? sendResponse(octokitResponse)
-      : await onUnhandledRequest(request);
+      : await onUnhandledRequest!(request);
```

Comment:
Because of L31, `onUnhandledRequest` should always be defined at this point, right?
```suggestion
      : await onUnhandledRequest(request);
```